Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md

```diff
@@ -0,0 +1,48 @@
+# Overview of Google Business Messages template flows
+
+- This document covers the purpose and configuration details for the below template flows for Google Business Messages channel
+  - Basic Inbound flow
+  - Inbound flow with suggestions
+- Inorder to publish the flows you will need assets configured for Google Business Messages channel you can find the documentation for asset configuration in https://help.imiconnect.io/docs/google-business-messages
+
+## Overview of Google Business Messages Inbound flow
```

Comment:
Can we call this as the basic inbound flow? Like we call the next one as Inbound flow with suggestions. Or vice versa to maintain consistency.